qframelesswindow/utils/win32_utils.py

```diff
@@ -20,10 +20,7 @@ def isMaximized(hWnd):
         window handle
     """
     windowPlacement = win32gui.GetWindowPlacement(hWnd)
-    if not windowPlacement:
-        return False
-
-    return windowPlacement[1] == win32con.SW_MAXIMIZE
+    return windowPlacement[1] == win32con.SW_MAXIMIZE if windowPlacement else False
```

Comment:
The reason why I write the return statement like this can be found in [Tips for cleaner code: Cleaning up IF statements](https://www.youtube.com/watch?v=ldqDpmMkXgw)